### ISSUE TITLE
Unable to post via chat API

### ISSUE THREAD

**Issue by gep13** [cake-contrib/Cake.Slack#25]
Trying to use this code:

```
var slackToken         = "blah";
var slackChannel        = "blah";

try
    {
        Information("Sending message to Slack...");

        var postMessageResult = Slack.Chat.PostMessage(
                    token: slackToken,
                    channel: slackChannel,
                    text: "this is a test"
            );

        if (postMessageResult.Ok)
        {
            Information("Message {0} successfully sent", postMessageResult.TimeStamp);
        }
        else
        {
            Error("Failed to send message: {0}", postMessageResult.Error);
        }
    }
    catch(Exception ex)
    {
        Error("{0}", ex);
    }
});
```

results in:

`Failed to send message: not_authed`

Seems like there was a breaking change in Slack API, as the token is no longer passed in JSON payload, but rather as part of the URL.

**Comment by jzeferino** [comment]
Any update on this? @gep13 @devlead

**Comment by jzeferino** [comment]
Please guys take a look at my PR #26.

**Comment by luisvitela** [comment]
When can we have a release for PR #26 ? @gep13 @devlead

**Comment by devlead** [comment]
Fixed by #26

**Comment by gep13** [comment]
@devlead thanks for getting this pulled in.
